v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+publisher: felixfbecker
+name: vscode-php-debug
+version: 1.13.0
+displayName: PHP Debugger
+title: PHP Debugger
+type: VS Code extension
+description: This VS Code extension provides support for PHP debugging, based on the PHP Debug extension from Felix Backer.
+icon: https://raw.githubusercontent.com/felixfbecker/vscode-php-debug/v1.13.0/images/logo.png
+repository: https://github.com/felixfbecker/vscode-php-debug
+category: Language
+firstPublicationDate: "2019-04-16"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-php:7-b95f23f"
+      name: php-debugger
```

Comment:
I wonder if this needs a memory limit

Reply:
What's the policy re: memory limits? I see the devfiles that use this plugin have one set.